ENH: provide a charter for DANDI (#2)
BICCN/BICCN-Infrastructure-Draft: DANDI.md

> # DANDI: Distributed Archives for Neurophysiology Data Integration

Reviewer: @tgbugs provide list of data modalities to @satra

Author: @tgbugs - let me know if there is a specific way to encode this.

## Description

> DANDI is a platform for publishing, sharing, and processing neurophysiology data funded by the BRAIN Initiative. We will work with BICCN and other BRAIN awardees to curate data using community standards, and to make data FAIR (Findable, Accessible, Interoperable, and Reusable). We will use DANDI to help improve our understanding of the brain.

Reviewer 2: Could you call out specific kinds of data you are interested in working with?

# Roles
- [Principal Investigator] Satrajit Ghosh (satra@mit.edu)
- [Principal Investigator] Yaroslav Halchenko (yoh@dartmouth.edu)
- [NWB and community liaison] Ben Dichter (ben.dichter@gmail.com)
- [Co-Investigator] Michael Grauer (michael.grauer@kitware.com)

> ## Definitions

Author: - add BIDS, NIDM
- add INCF BIDS endorsement link
- add URLs to these words

HDF - Hierarchical Data Format supports an unlimited variety of datatypes, and is designed for flexible, efficient I/O and for high volume and complex data.

NWB - Neurodata Without Borders. The NWB file specification can capture the entirety of heterogeneous data representing an experiment and is built on top of HDF. 

NWB:N - NWB:Neurophysiology, is a common file format that accommodates a wide variety of neurophysiology data as well as complex metadata related to stimulus and behavior. This is being developed under BRAIN award [1R24MH116922-01](https://projectreporter.nih.gov/project_info_description.cfm?aid=9582696).

## Objectives

> - Develop a cloud platform for versioned neurophysiology data storage for the purposes of collaboration, archiving, and preservation. 

Author: ```suggestion
- Develop a cloud platform for versioned neurophysiology data storage for the purposes of collaboration and dissemination of data and software containers. 
```

- Provide easy to use tools for neurophysiology data submission and access in the archive; and 
- Facilitate adoption of NWB via standardized applications for data ingestion, visualization and processing.

As an exercise, let’s assume you lose all the data in your lab. What would you want from the archive? Our hope is that your answer to this question, the necessary data and metadata that you need, is at least what we should be storing.

> Our goals for the archive, we think, are rather simple:

Reviewer 2: I am excited to learn about your aspirations and be a partner in making them happen but I feel the charter should focus on what work we are committed and not committed to doing.

> Our goals for the archive, we think, are rather simple:

Author: - remove this section
-

> - Optimize storage based on cost to produce and process.

Reviewer: ```suggestion
- Superoptimize storage based on cost to produce and process.
```

- Enable reproducible practices and publications.
- Enable secondary uses of data outside the intent of the study.
- Reduce the need to contact data producers by enriching the data with rich metadata.
- If users have to come back to the raw data, for example, to apply different processing strategies, etc., it should be feasible without going to the data producer.

> - The archive is not just an endpoint to dump data, it is intended as a living repository that enables collaboration within and across labs, and for others, the entry point for research.

Author: + metadata/data exchange with other archives, compute services, and coordinating centers

## In-scope

> - Develop infrastructure for ingesting, storing, searching across, and retrieving datasets containing cellular neurophysiology data and common data derivatives

Reviewer 2: Is interactivity to the BCDC portal in scope?

Author: added on in response to this question and @carolth comments.

- Develop web portal to support data and workflow management
- Support for BRAIN standards (NWB:N, BIDS, NIDM)
- Data availability with at least file level atomicity
- Provide human and programmatic search of data and metadata

> - Store derive data if it is of use to the community beyond a lab

Reviewer 2: Please define derived data in definitions.

> - Store derive data if it is of use to the community beyond a lab

Reviewer 2: ```suggestion
- Store derived data if it is of use to the community beyond a lab
```

> ## In-scope

Reviewer 3: For consideration

Support API to access indexed timeseries traces from select dataset to enable the visualization and analysis of timeseries in third party system such as the BCDC Cell Registry

Author: @lydiang - we will likely support this, but the api development will take a bit of time. @mgrauer, @yarikoptic @bendichter and i will be having meetings with people during SFN to guide the development of the API,  so this kind of feedback would be much appreciated.

> - Data modalities: multi-channel or single channel timeseries and 3D/4D imaging data

Author: @tgbugs - let me know how you would want this encoded. i think it should go in a predefined separate section.

- Provide software to capture experimental metadata, convert to NWB, and upload/download data to/from the archive
- Support browser-based visualization of data
- Provide tutorials and training for users to use the archive

> - We will allow limited storage of private data with an embargo period, after which data becomes public

Author: - when MRI or optical imaging enters this archive.

> - We will allow limited storage of private data with an embargo period, after which data becomes public

Reviewer 2: maybe change this to support the BICCN embargo policy (as the consortium defines it). That feels like a policy we will have listed here separately.

Author: i would like to discuss this today if possible. i am not sure this has been defined yet.

Reviewer 2: I added developing an Embargo Policy as an issue to the repo.

## Out-of-scope
- Develop neurophysiology standards
- Develop neurophysiology software

> - Provide unlimited computing resources

Author: - pure MRI datasets without any cellular level data
